File: container/public/container.cpp
#include "judge.h"

class Container : public IContainer {

public:
	 void init(int size) {
	 }
	 bool add(int a) {
	 }
	 bool contains(int a) {
	 }
	 bool remove(int a) {
	 }

};

int main() {
	Container * container = new Container();
	Judge::run(container);
}

File: src/main/java/bits/Container.java
package bits;

import java.util.BitSet;

public class Container {
    private BitSet bitSet;
    private int size;

    public void init(int size) {
        this.size = size;
        bitSet = new BitSet(size);
    }

    public boolean add(int a) {
        if (a < 0 || a >= this.size) {
            return false;
        }
        if (!bitSet.get(a)) {
            bitSet.set(a);
            return true;
        } else {
            return false;
        }
    }

    public boolean contains(int a) {
        if (a < 0 || a >= size) {
            return false;
        }
        return bitSet.get(a);
    }

    public boolean remove(int a) {
        if (a < 0 || a >= size) {
            return false;
        }
        if (bitSet.get(a)) {
            bitSet.clear(a);
            return true;
        } else {
            return false;
        }
    }
}
